build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import java.util.Base64
 
 allprojects {
-    version = "1.0.0-SNAPSHOT"
+    version = "2.0.0-SNAPSHOT"
```

Comment:
Should there not be a 1.0.0 before moving to 2.0.0-SNAPSHOT? Or there is one? :)

Reply:
1.0.0 is already released

Reply:
OK, shouldn't 1.0.0-SNAPSHOT come before 1.0.0? Not that it matters though, let's move on :)